main.go

```diff
@@ -59,7 +60,7 @@ const infoDirectory = "./static/bidder-info"
 func loadConfig(bidderInfos config.BidderInfos) (*config.Configuration, error) {
 	v := viper.New()
 	config.SetupViper(v, configFileName, bidderInfos)
-	return config.New(v, bidderInfos)
+	return config.New(v, bidderInfos, openrtb_ext.NormalizeBidderName)
```

Comment:
Do we need to pass `openrtb_ext.NormalizeBidderName` from `main.go` all the way down to `config/bidderinfo.go`'s `applyBidderInfoConfigOverrides(...)` function? I see the need for `bidderInfos` to be passed from `main.go` and also the convenience of passing a mock function to `applyBidderInfoConfigOverrides(...)` for testing purposes but, I'm wondering if maybe we can simplify `config.New(...)` by not adding that third paramenter:
```
619   // New uses viper to get our server configurations.
620 - func New(v *viper.Viper, bidderInfos BidderInfos) (*Configuration, error) {
    + func New(v *viper.Viper, bidderInfos BidderInfos, normalizeBidderName func(string) (openrtb_ext.BidderName, bool)) (*Configuration, error) {
621       var c Configuration
622       if err := v.Unmarshal(&c); err != nil {
623   *-- 78 lines: return nil, fmt.Errorf("viper failed to unmarshal app config: %v", err)------------------------------------
701       resolvedStoredRequestsConfig(&c)
702   
703 -     mergedBidderInfos, err := applyBidderInfoConfigOverrides(c.BidderInfos, bidderInfos, normalizeBidderName)
    +     mergedBidderInfos, err := applyBidderInfoConfigOverrides(c.BidderInfos, bidderInfos, openrtb_ext.NormalizeBidderName)
704       if err != nil {
705           return nil, err
706       }
config/config.go
```
Therefore we could rollback to:
```
60   func loadConfig(bidderInfos config.BidderInfos) (*config.Configuration, error) {
61       v := viper.New()
62       config.SetupViper(v, configFileName, bidderInfos)
63 -     return config.New(v, bidderInfos, openrtb_ext.NormalizeBidderName)
   +     return config.New(v, bidderInfos)
64   }
main.go
```
Unless passing it from `main.go` makes something happen behind the scenes that I'm missing.

Reply:
It is necessary for `New` to accept the normalize function so unit tests are free to use fake bidder names instead of being forced to use real ones. The bidder name flow will continue to evolve and in the near future the normalize func will likely no longer be required. We're focused on restoring functionality to fix the bug and will design a better solution as a follow-up.

Reply:
Yes, I was thinking about this too. We also need this parameter in config_test.go -> `newDefaultConfig`